tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -253,16 +254,16 @@ def _run(self):
                     # volume_name": "vol1"}]}\n"
 
                     out_dict = json.loads(stdout[stdout.index('{'): -1])
-                    NS.gluster.objects.GlobalDetails(
-                        status=out_dict['status']
-                    ).save()
                     NS.gluster.objects.Utilization(
                         raw_capacity=out_dict['raw_capacity'],
                         usable_capacity=out_dict['usable_capacity'],
                         used_capacity=out_dict['used_capacity'],
                         pcnt_used=(out_dict['used_capacity'] * 100 / out_dict['usable_capacity'])
                     ).save()
+                    volume_up_degraded = 0
                     for item in out_dict['volume_summary']:
+                        if "up(degraded)" in item['state']:
```

Comment:
@r0h4n yes it present, "state": "up" or "state": "up(partial) " or "state": "up(degraded) "  or "state": "down"
four states are available in gstatus -o json command